### ISSUE TITLE
Clicking "View image" opens blank page

### ISSUE THREAD

**Issue by kpershin** [bijij/ViewImage#207]
I see blank page when I'm trying to open full size image

Steps
1. Open any image in Google
2. Click "View Image"
3. See blank page

Pics
1.
![image](https://user-images.githubusercontent.com/60386587/73256833-b6dc0980-41d3-11ea-8da7-1e7c98ef3889.png)

2. Opens this
![image](https://user-images.githubusercontent.com/60386587/73256909-d83cf580-41d3-11ea-8d8d-b7b3ae0ed0dd.png)

––––––––––––––––
Mac OS Catalina
Chrome 76.0.3809.132
ViewImage v3.2.2 (mac x86-64) - ru

**Comment by JustusGS** [comment]
Same issue occurring for me. I assume Google changed something?

MacOS Catalina 10.15.2
Chrome 79.0.3945.130
v3.2.2 (mac x86-64) - en_US

**Comment by SteveBaker** [comment]
Yeah - me too - on all platforms.   Looks like Google did something.

**Comment by bijij** [comment]
At the moment I can't repro unfortunately.
likely A:B testing on Google's end 😞

**Comment by ikeblaster** [comment]
@bijij I can confirm the issue. It looks like google is creating some temporary container with base64 image and then swaps it with new container which contains the image from real URL. But not always - if you switch image below preview (in right pane), then it just swaps img `src` attribute. Using mutation observer on the `.n3VNCb` (which are image previews) attributes seems to be reliable in both cases.

Please can you test this fixed version on your end? I wonder whether class `tvh9oe` was already present in OCT19 or we need to split it into JAN20. Sorry for not creating PR, too much hassle for one file when I don't know if works for everyone. :/

https://pastebin.com/h5GgVUnH

Tested in chrome and FF.

**Comment by bijij** [comment]
This seems to work for me, I'm not sure if `tvh9oe` always existed or not.

I'll be happy to commit this and push a new version if you're okay with that.

**Comment by bijij** [comment]
sorry @ikeblaster forgot to mention you.

**Comment by ikeblaster** [comment]
@bijij Of course, please do.

**Comment by bijij** [comment]
Gone ahead and published a copy with those changes, lets see if it fixes it for people.

**Comment by Leon422** [comment]
@bijij I don't see any updates. Is it not published yet on the addon store?

**Comment by bijij** [comment]
@Leon422 what's version is your extension currently?

**Comment by Leon422** [comment]
@bijij v3.2.2 (win undefined) - undefined. I'm on Firefox. The addon store says the extension was last updated 3 months ago.

**Comment by bijij** [comment]
@Leon422 I repushed to AMO should be live in a bit.

**Comment by Leon422** [comment]
@bijij It finally updated, and it's working fine again. Thanks!

**Comment by MrNotSoNice** [comment]
Thanks!  It's working!

**Comment by kpershin** [comment]
Thanks @bijij It’s working!

**Comment by needforsuv** [comment]
It still shows the thumbnail if you click view image for larger images where only the thumbnail loads

**Comment by needforsuv** [comment]
if the main image isn't one of the main ones

**Comment by tsuliwaensis** [comment]
I'm using v3.2.2 of the UserScript version on Chrome 81.0.4044.43, and this is still broken.

**Comment by JohannesMP** [comment]
This issue is still the case and seems to be occurring more frequently

**Comment by JohannesMP** [comment]
The 'Google Images Restored' extension appears to not suffer from this issue - https://chrome.google.com/webstore/detail/google-images-restored/ncndcebmkibkhopclfdjfacgfholcghi/related

Specifically for a given image result where with the 'View Image' extension exhibits this bug, viewing the same search and having the same search result selected when instead having the 'Google Images Restored' extension installed will not cause this bug, and all images seem to both render their previews as well as as open the image as desired.

For the time being I would therefore recommend anyone that is encountering this issue with the 'View Image' extension to consider 'Google Images Restored' instead.

I have no affiliation either way, but this issue seems to have remained unfixed for just about two years now.
